src/module.ts

```diff
@@ -2,9 +2,41 @@ import defu from 'defu'
 import { resolve } from 'pathe'
 import { defineNuxtModule, addPlugin } from '@nuxt/kit'
 import type { Nuxt } from '@nuxt/schema'
-import type { StrapiOptions } from './types'
+import type { CookieOptions } from 'nuxt3/dist/app/composables/cookie'
 
-export default defineNuxtModule<StrapiOptions>({
+export interface ModuleOptions {
+  /**
+   * Strapi API URL
+   * @default process.env.STRAPI_URL
+   * @example 'http://localhost:1337'
+   * @type string
+   */
+  url?: string
+
+  /**
+  * Strapi Prefix
+  * @default '/api'
+  * @type string
+  */
+  prefix?: string
+
+  /**
+   * Strapi Version
+   * @default 'v4'
+   * @type string
+   * @example 'v3'
+   */
+  version?: 'v4' | 'v4'
```

Comment:
Hi @pi0

Shouldn't this be `'v4' | 'v3'`?

Reply:
@luke-z Thanks for the heads up I missed this, fixed in 6ad2e38ec4434db1d0dd1342502322eebd845948.

Reply:
Awesome :)